File: basic/Question1.r
#Question1: Accept two int values from user and return their product.
#if the product is greater than 1000, then return their sum.

args <- commandArgs(trailingOnly = TRUE)

a <- as.integer(args[2])
b <- as.integer(args[3])

producto <- a*b

if (producto > 1000){
	suma <- a+b
	cat("La suma es", suma, "\n")
}else{
	cat("El producto es", producto, "\n")
}

File: basic/Question1.py
#Question1: Accept two int values from user and return their product.
#if the product is greater than 1000, then return their sum.

#Puedo usar la función input para esperar que el usuario ingrese un String.
import sys

a = int(sys.argv[1])
b = int(sys.argv[2])
producto = a*b
if (producto > 1000):
	suma = a+b
	print("la suma es", suma)
else:	
	print("el producto es", producto)